code/game/atoms_movable.dm

```diff
@@ -754,4 +754,92 @@
 	appearance = falltempoverlays[src]
 	falltempoverlays -= src
 	ignoreinvert = initial(ignoreinvert)
-	invisibility = init_invisibility
\ No newline at end of file
+	invisibility = init_invisibility
+
+/datum/proc/send_to_past(var/duration)
+	return
+
+/datum/var/being_sent_to_past = FALSE
+
+/atom/movable/send_to_past(var/duration)
+	var/current_loc = loc
+	var/list/resettable_vars = list(
+									"being_sent_to_past",
```

Comment:
I figured the standard for lists was intending out to the end of the line where the list is defined.
I can back it up to just one or two tabs if it's a problem.